src/NuGet.Services.EndToEnd/Support/TestSettings.cs

```diff
@@ -28,86 +31,148 @@ public enum Mode
         /// </summary>
         public static bool DefaultAggressivePush => true;
 
+        /// <summary>
+        /// A list of default test settings per environment.
+        /// </summary>
+        public static readonly Dictionary<Mode, TestSettings> DefaultTestSettings = new Dictionary<Mode, TestSettings>
```

Comment:
Public static dictionary is a bad practice as they are mutable. This should be a method or an read only dictionary.